internal/pkg/runtime/engine/config/starter/starter_linux.go

```diff
@@ -200,6 +200,18 @@ func (c *Config) SetAllowSetgroups(allow bool) {
 	}
 }
 
+// SetNoSetgroups disables the setgroups call for the container process in the
+// starter. Preserves access to files that depends on supplementary groups
+// outside of the user namespace. The supplementary groups will map to 'nobody'
+// inside the container.
+func (c *Config) SetNoSetgroups(allow bool) {
```

Comment:
I found the name `allow` for the argument here a bit hard to grok. Especially since, in this context, passing `true` _prevents_ something from happening (namely, a setgroups call).